src/main/java/frc/robot/commands/JoystickTarget.java

```diff
@@ -35,10 +36,16 @@ public JoystickTarget(DriveSubsystem drive, Limelight limelight, GenericHID driv
   public void initialize() {
     thetaController.reset(0);
     thetaController.setTolerance(VisionConstants.kTargetingTolerance);
+    SmartDashboard.putNumber("Max Angular Speed", SmartDashboard.getNumber("Max Angular Speed", SwerveModuleConstants.kMaxAngularSpeedRadiansPerSecond));
+    SmartDashboard.putNumber("Max Angular Acceleration", SmartDashboard.getNumber("Max Angular Acceleration", SwerveModuleConstants.kMaxAngularAccelerationRadiansPerSecondSquared));
+    SmartDashboard.putNumber("kP", SmartDashboard.getNumber("kP", SwerveModuleConstants.kPTurningController));
+    SmartDashboard.putNumber("kI", SmartDashboard.getNumber("kI", SwerveModuleConstants.kITurningController));
+    SmartDashboard.putNumber("kD", SmartDashboard.getNumber("kD", SwerveModuleConstants.kDTurningController));
```

Comment:
SmartDashboard quickly becomes overwhelmingly cluttered if we commit diagnostics stuff like this. I would recommend enclosing this code in an `if (false) {...}` block so that it's easy to enable during testing, but it's not always enabled. Alternatively, if you want this always enabled for the time being, an `if (true) {...}` block would be fine. (Same thing goes for other SmartDashboard stuff in this pull request.)

Reply:
I switched to posting directly to network tables with a "tuning" prefix, so we can pull them up only when we want.